.travis.yml

```diff
@@ -16,15 +16,19 @@ cache:
   yarn: true
   directories: node_modules
 
+env:
+  global:
+    - REACT_APP_API_URL="http://localhost:8000/api"
+
 script:
   - cd $TRAVIS_BUILD_DIR/code/datalab-app/ && yarn install && yarn test && yarn lint
 
 after_success:
   - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
       yarn global add gitbook-cli;
-      cd $TRAVIS_BUILD_DIR/code/datalab-app/ && yarn build && docker build -t nerc/datalab-app .;
+      cd $TRAVIS_BUILD_DIR/code/datalab-app/ && yarn build && docker build -f app.Dockerfile -t nerc/datalab-app . && docker build -f api.Dockerfile -t nerc/datalab-api .;
```

Comment:
Fine for this PR, but I think this is getting to the limit of acceptable complexity within the CI server script. Next time we change this I think the whole lot needs to be pulled out into a standalone bash script as that will allow local testing.